<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.sandata.v8.functional.Exports.Auto_V7_TC_64319_Validate_Dept_Tasks" tests="1" time="29.635" ignored="0" failures="0" timestamp="07 Apr 2017 18:56:52 GMT" skipped="0" hostname="DESKTOP-KE0VAOI" errors="0">
  <testcase name="Auto_V7_TC_63092_Error_With_Date_Range_365_Days" time="29.635" classname="com.sandata.v8.functional.Exports.Auto_V7_TC_64319_Validate_Dept_Tasks"/>
</testsuite> <!-- com.sandata.v8.functional.Exports.Auto_V7_TC_64319_Validate_Dept_Tasks -->
